docs/deploying/deploy-nextjs-app.mdx

```diff
@@ -33,6 +35,25 @@ If you omit the `--prod` flag it will deploy it to a preview/test URL.
 
 **Make sure to check the values of your Scaffold Configuration before deploying your NextJS App.**
 
+## Deploy to BG IPFS
+
+You can also deploy your app to BG IPFS by running:
+
+```
+yarn ipfs
+```
+
+This will build your app and deploy it to BG IPFS. After deployment, you'll receive a URL where your app is accessible.
+
+Below is an example of the output you'll see:
+
+```
+🚀 Upload complete! Your site is now available at: https://community.bgipfs.com/ipfs/bafybeicrhakhu4bkzyuuaprtie4dkup6wpiplndnsisd2gzv4uzayurvem
+```
+
+And that is all!
+If you want to use your ENS name as a gateway(e.g. `your-ens-name.eth.limo` will point to your app), you can check the following [guide](https://eth-limo.gitbook.io/documentation/beginner/configuring-your-ens-name/updating-your-ens-content-records).
```

Comment:
I think we can remove this part completely 

1. Like if we update the bgipfs npm package and it changes the output we need to remember to update it here 
2. Regarding connecting to ENS name I think that's more of general instead something special you need to do with bgipfs to setup it like that...so people shall be able to find it easily 🙌